src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidationImpl.java

```diff
@@ -83,27 +98,17 @@ public ValidateApplicantForSharingResponse validateApplicantForSharing(
 	}
 	
 	@Override
-	public ValidateApplicantPartyIdResponse validateApplicantPartyId(String applParyId)
+	public ValidateApplicantPartyIdOrdsResponse validateApplicantPartyId(String applPartyId)
 			throws FigaroValidationServiceException {
-		
-		// TODO - replace the following dummy response with a call to ORDS  
-		
-		// Note: Call response code and response message comes from the ORDS call.  
-						
-		ValidateApplicantPartyIdResponse resp = new ValidateApplicantPartyIdResponse(
-					"success",
-					0, 
-					"Cool",
-					"Joe", 
-					"Second",
-					"2016/10/14",
-					"0123456",
-					"Toronto",
-					"F"
-					); 
-			
-			return resp;
-			
+
+		try {
+			return ordsapi.validateApplicantPartyId(applPartyId, null);
+		} catch (ApiException ex) {
+			logger.error("Exception caught as Figaro Validator Service, ValidatePartyId : " + ex.getMessage()); 
+			ex.printStackTrace();
```

Comment:
I like this pattern, with only the logger going to splunk, we will have the stack trace on openshift only. nice